docs/source/contributing.rst

```diff
@@ -2,49 +2,43 @@
 Contributing to GTG
 ===================
 
-GTG uses Bazaar_ for versioning. It might be useful to read `Bazaar's tutorial`_ first.
+GTG uses GitHub_ for versioning. It might be useful to read `GitHub's Guide`_ first.
 
-.. _Bazaar: http://bazaar.canonical.com/
-.. _`Bazaar's tutorial`: http://doc.bazaar.canonical.com/latest/en/mini-tutorial/
+.. _GitHub: https://github.com/
+.. _`GitHub's Guide`: https://guides.github.com/
 
 Dependencies
 ============
 
-You need to have python-configobj installed
+You need to have python3 installed. For complete info look at README_.
+
+.. _README: https://github.com/getting-things-gnome/gtg/blob/master/README.md
 
 Getting the code
 ================
 
-Get the latest version of the code on Launchpad_::
-
-    $ bzr branch lp:gtg trunk
-
-Although if you're thinking of contributing more than one patch, you might want to do::
-
-    $ bzr init-repo gtg
-    $ cd gtg
-    $ bzr branch lp:gtg trunk
+Get the latest version of the code on GitHub_. We suggest forking the master branch at first.
+Then clone the forked master to your local::
 
-This will share revision data between branches, reducing storage costs & network time.
+    $ git clone https://github.com/YOUR_GITHUB_USERNAME/gtg.git
 
+Launch GTG with debugging data (so it doesn't mess with your data)::
 
-Launch gtg with debugging data (so it doesn't mess with your data)::
+    $ cd path/to/gtg
+    $ ./gtg.sh
 
-    $ cd trunk
-    $ ./scripts/debug.sh
-
-.. _Launchpad: https://launchpad.net
+.. _GitHub: https://github.com/getting-things-gnome/gtg
 
 Choosing a feature to work on
 =============================
 
 If you are a happy user of GTG and nothing bothers you but you would like to contribute you can:
 
-* choose a `LOVE bug`_ which are easier to solve
+* choose a bug from our `Bugs list`_ and try to solve
 * ask people on IRC channel #gtg on irc://irc.gimp.org/#gtg
 * ask on our `mailing list`_
 
-.. _`LOVE bug`: https://bugs.launchpad.net/gtg/+bugs?field.status%3Alist=NEW&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&field.status%3Alist=INPROGRESS&assignee_option=none&field.tag=love
+.. _`Bugs list`: https://github.com/getting-things-gnome/gtg/issues
```

Comment:
Point rather to our LOVE bugs: https://github.com/getting-things-gnome/gtg/labels/love